astro/src/content/docs/get-started/core-concepts/roles.mdx

```diff
@@ -131,7 +131,6 @@ _FusionAuth application roles_
 
 The `user_support_manager` role is a role tuned for tier 1 technical support personnel and has a mix of capabilities. A user with such a role can:
 
-_FusionAuth user_support_manager_
```

Comment:
Did you mean to remove this heading?

Reply:
I did because it seemed a bit redundant on the page, there were essentially two headings. But I can add it back as I see we do that a decent amount through our docs. The other piece was the real change that needed to be made. I can re-submit with that put back in there.

Reply:
Put it back, thanks.